sonarqube/content.md

```diff
@@ -42,14 +42,18 @@ We recommend creating volumes for the following directories:
 -	`/opt/sonarqube/conf`: **for Version 7.9.x only**, configuration files, such as `sonar.properties`.
 -	`/opt/sonarqube/data`: data files, such as the embedded H2 database and Elasticsearch indexes
 -	`/opt/sonarqube/logs`: contains SonarQube logs about access, web process, CE process, Elasticsearch logs
--	`/opt/sonarqube/extensions`: plugins, such as language analyzers
+-	`/opt/sonarqube/extensions`: for all 3rd party plugins 
```

Comment:
From what I know, all plugins are 3rd party now. That's why we had this wording.

Reply:
then we might just want to get rid of the `language analyzers` part? seems confusing to me

Reply:
indeed, it can be confusing.

```suggestion
-	`/opt/sonarqube/extensions`: 3rd party plugins 
```